packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -1005,9 +1025,15 @@ contract Deploy is Deployer {
             )
         });
 
-        SystemConfig config = SystemConfig(systemConfigProxy);
-        string memory version = config.version();
-        console.log("SystemConfig version: %s", version);
+        if (cfg.useInterop()) {
+            SystemConfigInterop config = SystemConfigInterop(systemConfigProxy);
+            string memory version = config.version();
```

Comment:
This diff is also not necessary there is a logline elsewhere that says its interop enabled